failure-summary.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Finds failing Checks and combines each help message into an array
+ * @param  {Object} nodeData Individual "detail" object to generate help messages for
+ * @return {String}          failure messages
+ */
+var failureSummary = function failureSummary(nodeData) {
+	'use strict';
+
+	var failingChecks = {};
+	// combine "all" and "none" as messaging is the same
+	failingChecks.none = nodeData.none.concat(nodeData.all);
+	failingChecks.any = nodeData.any;
+
+	return Object.keys(failingChecks).map(function (key) {
```

Comment:
ok, I see the message

Here is what it looks like in the example on my machine

![image](https://cloud.githubusercontent.com/assets/79245/18789197/1ae51392-8178-11e6-9ea9-441de8a57b09.png)

The order of the messages is confusing. It should go:

Element (group)
->HTML
->Failure Summary (group)
->->Related nodes for failure summary

WDYT?